src/Spice86.Core/Emulator/InterruptHandlers/VGA/Records/CursorPosition.cs

```diff
@@ -0,0 +1,6 @@
+namespace Spice86.Core.Emulator.InterruptHandlers.VGA.Records;
+
+/// <summary>
+///     Represents a cursor position.
+/// </summary>
+public record struct CursorPosition(int X, int Y, int Page);
```

Comment:
Missing documentation for Page. What is a page in this context ?

Reply:
Docs added